clients/java/signalr/build.gradle

```diff
@@ -36,16 +36,30 @@ task generatePOM {
             version '0.1.0-preview1'
```

Comment:
There's a `groupId` field on line 34 that's incorrect. I think you don't need to specify `groupId`, `artifactId`, or `version` and it will pull them from the Gradle top-level properties. I did a quick test removing them and ran `gradlew generatePOM` and the generated POM still had the right values. That would let us remove duplication :).